FunctionExecutor.gd

```diff
@@ -92,7 +92,8 @@ func tests_all_functions() -> void:
 						to_print += ")"
 						print(to_print)
 
-					object.callv(method_data["name"], arguments)
+					var ret = object.callv(method_data["name"], arguments)
+					BasicData.remove_thing(ret)
```

Comment:
This doesn't respect disabled classes.
For now this try to remove e.g MainLoop which is returned by `get_main_loop` and is on list disabled classes.
This also should be disabled conditionally in regression_test_project(additional variable needs to be added at the top of script)

Reply:
Not sure what you mean by adding additional variable at the top of the script. If I understand you correctly, see new changes below.